public/components/Glossary/Glossary.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const MedianAttentionTimeExplainerText = 'Attention time is the value where a user is knowingly active on the page; they have recently moved their mouse or scrolled.';
+const DwellTimeExplainerText = 'Dwell time tries to emulate the same viewability metric from Google Analytics. It is calculated by taking the difference between a pageview timestamp and the subsequent pageview timestamp. These pageviews have to happen within a 30 minute session period, and are averaged when there has been more than one pageview for a unique user.';
```

Comment:
Perhaps `as Google` instead of `from Google`?

Reply:
I think `from` makes more sense? I don't understand exactly what `as` would mean! I think it might not even be called dwell time in GA either, it might be time on page... Need to check.